### ISSUE TITLE
NAT Instance?

### ISSUE THREAD

**Issue by jplock** [alexdebrie/serverless-vpc-examples#1]
For small applications, a [NAT Instance](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_NAT_Instance.html) may be an ok option if people want to save money and not pay for a NAT Gateway.

https://github.com/smoketurner/serverless-vpc-plugin supports provisioning NAT instances if you wanted to leverage that plugin in any way.

**Comment by alexdebrie** [comment]
Good point, @jplock, and good to see you again! I thought about including this but held off. You're right that I should mention it as an option.

Will leave this one open but aim to add it in the next few weeks.
